bk-api/test_bk_api.py

```diff
@@ -139,6 +139,30 @@ def test_log_insert_fail(client):
     assert 'error'  in result
 
 
+def test_vsn_insert(client):
+    #Check that the initial value is null
+    rv = client.get(f'/state/0000000000000001')
+    result_before_vsn = rv.get_json()
+    d_before_vsn = result_before_vsn["data"]["vsn"]
+    assert 'error' not in result_before_vsn
+    assert 'data' in result_before_vsn
+    assert None == d_before_vsn
+    #Check that we can populate vsn
+    data = {"vsn": True}
+    rv = client.post('/node/0000000000000001', data = json.dumps(data))
+    result = rv.get_json()
+    assert "success" in result
+    #Check that the db is updated with the new value
+    gt_value = "TEST-minimal"
```

Comment:
why "TEST-minimal" ?  where does this come from?  how does node `0000000000000001` have a VSN of "TEST-minimal"?  Where did you define that on your test node?

Reply:
I agree totally agree.

At some point, we started putting some Vagrant stuff there to help run it, so the repo became less "pure" k8s stuff.

I'd prefer to split.